### PR TITLE
Use fonts 2.0 that don't require wrapping codepoints

### PR DESCRIPTION
**Pull request by gkatsev**
## Description
The title says it all

### CODE REVIEW

File: src/css/vjs.scss [hunk 1]
```diff
@@ -1,3 +1 @@
-$icon-codepoints: true;
```
Comment (gkatsev) [inline src/css/vjs.scss]: Removing this file entirely would technically be a breaking change, even removing `icon-codepoints` could be questionable but I don't think many people have been using it.

Comment (nickygerritsen) [inline src/css/vjs.scss]: Agreed, don't really think this is a problem

**Comment by nickygerritsen** [comment]
LGTM, makes the code a lot cleaner :)

**Comment by misteroneill** [comment]
Very nice, LGTM. Agree that while it could be construed as a (very edge-casey) breaking change, it's not significant enough to merit a 6.0 👍

**Comment by gkatsev** [comment]
I'll just revert the change if anyone complains.

**Pull request closed without merging**